components/brave_shields/browser/ad_block_subscription_service_manager.cc

```diff
@@ -326,6 +349,35 @@ void AdBlockSubscriptionServiceManager::OnGetDownloadManager(
       base::DoNothing());
 }
 
+void AdBlockSubscriptionServiceManager::OnListMetadata(
+    const GURL& sub_url,
+    const adblock::FilterListMetadata& metadata) {
+  // The engine will have loaded new list metadata; read it and update local
+  // preferences with the new values.
+
+  DCHECK_CALLED_ON_VALID_SEQUENCE(sequence_checker_);
+
+  auto info = GetInfo(sub_url);
```

Comment:
I believe you are calling GetInfo on more than one thread now. Please add a thread check to it